edgelet/edgelet-http/src/error.rs

```diff
@@ -75,24 +83,33 @@ pub enum ErrorKind {
     #[fail(display = "Module not found")]
     ModuleNotFound(String),
 
+    #[fail(display = "A native TLS error occurred.")]
+    NativeTls,
```

Comment:
Replace this and `OpenSSLPkcs12` below with a kind corresponding to the *operation* that this error is raised from. See 326ef8c0075e66834c82b36fc5551b8ef74f0098 for details.